samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -173,152 +247,130 @@ class DailyCallManager {
+  updateVideoUi(track, participantId) {
     let videoContainer = document.getElementById(
       `video-container-${participantId}`
     );
-
-    // If a container for this participant doesn't exist, create it
     if (!videoContainer) {
       videoContainer = document.createElement('div');
-      videoContainer.id = `video-container-${participantId}`; // Use participant ID for unique container ID
+      videoContainer.id = `video-container-${participantId}`;
       videoContainer.className = 'video-container';
-      videoContainer.style.backgroundColor = '#ccc'; // Set a gray background for the placeholder
       document.getElementById('videos').appendChild(videoContainer);
-
-      const sessionIdText = document.createElement('div');
-      sessionIdText.className = 'session-id-overlay';
-      sessionIdText.textContent = participantId;
-      videoContainer.appendChild(sessionIdText);
     }
 
-    // Check if event track is active or not
-    if (!event.track.enabled) {
-      if (videoContainer.querySelector('video')) {
-        // If the video track is disabled but video element exists, clean up
-        videoContainer.querySelector('video').srcObject = null; // Free up media resources
-        // You might also want to change the placeholder style or add text/icon to indicate camera is off
-      }
-      return; // Exit without further processing if track is not active
+    let sessionIdOverlay = videoContainer.querySelector('.session-id-overlay');
+    if (!sessionIdOverlay) {
+      sessionIdOverlay = document.createElement('div');
+      sessionIdOverlay.className = 'session-id-overlay';
+      sessionIdOverlay.textContent = participantId;
+      videoContainer.appendChild(sessionIdOverlay);
     }
 
-    // Create new video element and add to container if not exists or reuse existing one
-    const videoEl =
-      videoContainer.querySelector('video') || document.createElement('video');
-    videoEl.className = 'video-element';
-    if (!videoEl.parentElement) {
+    let videoEl = videoContainer.querySelector('video.video-element');
+    if (!videoEl) {
+      videoEl = document.createElement('video');
+      videoEl.className = 'video-element';
       videoContainer.appendChild(videoEl);
     }
 
-    videoEl.style.width = '100%'; // Ensure the video occupies the full container width
-    videoEl.srcObject = new MediaStream([event.track]);
-    videoEl.play();
-  }
-
-  /**
-   * Adds an <audio> element to the DOM and attaches the audio track to it.
-   * @param {Object} event - The track event object containing the audio track.
-   */
-  playAudio(event) {
-    if (event.participant.local) {
-      return; // Avoid playing local audio
+    switch (track.state) {
+      case 'playable':
+        videoContainer.style.display = '';
+        videoEl.style.display = '';
+        break;
+      case 'off':
+      case 'interrupted':
+      case 'blocked':
+        videoEl.style.display = 'none'; // Hide video but keep container
```

Comment:
nit: so if `videoContainer` should _always_ have `.style.display = ''` regardless of track state, don't bother setting it in this `switch` case, above